### PR TITLE
fix logging

### PR DESCRIPTION
**Pull request by anthony-murphy**

### CODE REVIEW

**Review by markfields — APPROVED** [review]
🎂

Follow-up -- @jcmoore can you write some UTs to cover this and the rest of your change? :)

**Comment by anthony-murphy** [comment]
i was literally adding a ut for this case :) justin did add a bunch of tests here

File: packages/loader/container-utils/src/test/error.spec.ts [hunk 1]
```diff
@@ -15,6 +15,12 @@ describe("Errors", () => {
...
         });
+        it("Should preserve the stack", () => {
+            const originalError = new Error();
+            const testError = CreateContainerError(originalError);
+
+            assert(testError["stack"] === originalError.stack);
+        });
     });
 
     describe("DataProcessingError coercion", () => {
```
Comment (jcmoore) [inline packages/loader/container-utils/src/test/error.spec.ts:26]:
```suggestion
    describe("DataProcessingError coercion", () => {
    it("Should preserve the stack", () => {
            const originalError = new Error();
            const testError = CreateProcessingError(originalError);

            assert(testError["stack"] === originalError.stack);
        });
```

For now -- I'll move `extractLogSafeErrorProperties` and `extractLogSafeMessageProperties` into a telemetry-utils and write tests there.

**Review by curtisman — APPROVED** [review]
:shipit:

**Pull request merged**